docs/content/functioneel/informatieobjecten_api.md

```diff
@@ -0,0 +1,78 @@
+INFORMATIEOBJECTEN  API
+==========
+**FUNCTIONELE DOCUMENTATIE**
+
+
+| **Aspect**      | **Beschrijving**                                                                                                                                                                       |
+|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Doel            | Het opvragen van (lijsten van) enkelvoudige informatieobjecten en/of zaakinformatieobjecten |
+| Domein          | Zaakgericht werken.                                                                                             |
+| Provider        | [Documentregistratiecomponent](https://www.gemmaonline.nl/index.php/GEMMA2/0.9/id-0e99ec6c-283a-4ec9-8efa-e11468e6b878) |
+| Consumer        | [Documentbeheercomponent](https://www.gemmaonline.nl/index.php/GEMMA2/0.9/id-25ee9ea7-be66-4bdd-b40c-191777a88b35)  |
+| Informatiemodel | RGBZ2  |
+| Specificaties   | https://ref.tst.vng.cloud/drc/api/v1/schema/ |
+| Bijzonderheden  | - |
+
+Deze API omvat (vooralsnog) de resources Enkelvoudiginformatieobject en Zaakinformatieobject.
```

Comment:
Dit is outdated - de resources zijn `Enkelvoudigeinformatieobject` en `ObjectInformatieObject`.

Reply:
Op de master staat nog zaakinformatieobject: https://github.com/VNG-Realisatie/gemma-zaken/blob/master/api-specificatie/drc/openapi.yaml

Reply:
Ja, komt doordat #412 blijft liggen... :disappointed:

Reply:
Ik heb het alvast aangepast aangezien vorige week #412 is gemerged.